papers/emily_dorne/main.md

```diff
@@ -160,23 +160,31 @@ The [model experimentation](#model-experimentation) phase did not explore altern
+    <td>Similarly, climate and elevation features primarily provided value for data points prior to the launch of Sentinel-2 and so are not used in the final CyFi model. Climate and elevation likely do have an impact on how cyanobacteria blooms form, and more sophisticated feature engineering with these data sources may add value in the future. This is a direction for <a href="#future-directions">future research</a>.</td>
   </tr>
   <tr>
     <td>Incorporate land cover</td>
-    <td>Lastly, including a land cover map, even at a coarse 300m resolution, added to model accuracy. The land cover map likely captures farmland areas with fertilizer runoff that contributes to blooms. A static map from 2020 is used rather than a real-time satellite-derived product, as this reduces the compute time and patterns in land use do not fluctuate daily. Land cover is also an effective balance between reflecting regional characteristics, and avoiding overfitting to the small number data providers in the data. See the <a href="#generalizability-deep-dive">Generalizability deep dive</a> section for details.</td>
+    <td>Lastly, including a land cover map, even at a coarse 300m resolution, added to model accuracy. The land cover map likely captures farmland areas with fertilizer runoff that contributes to blooms. A static map from 2020 is used rather than a real-time satellite-derived product, as this reduces the compute time and patterns in land use do not fluctuate daily. Land cover is also an effective balance between reflecting regional characteristics, and avoiding overfitting to the small number data providers in the training set (see below for additional details).</td>
   </tr>
 </table>
 :::
 
+One of the risks in a machine learning competition is overfitting to the test set. Competition models may pick up on patterns specific to the competition data, rather than patterns of environmental cyanobacteria conditions that hold outside of the competition. The experimentation phase worked to identify and remove competition artifacts that would hamper the generalizability of the model in an open source package. For example, all winning solutions used a "longitude" feature in their models, which captured some underlying differences in sampling procedures by the 14 data providers for the competition. For example, California only samples cyanobacteria density for suspected blooms, leading to an over-representation of high density samples among competition data points in California. Predicting high severity for all values in California improves competition performance, but would not hold in the real world. As a result, geographic features like longitude, state, and region were not used for the deployed CyFi model.
```

Comment:
This discussion focuses on decisions that in the "data decisions" table, so moving it here situates it more clearly within the rest of the section. Otherwise, it looks like it's part of the "target variable" section.

Reply:
good call